live-examples/js-examples/array/array-flat.js

```diff
@@ -0,0 +1,11 @@
+const arr1 = [0, 1, 2, [3, 4]];
+
+// Flatten to single level array.
```

Comment:
Generally we don't use comments in interactive examples except for "expected output" or exceptional non-obvious things. I suppose it's a kind of "show don't tell" kind of thing - we have a whole page of docs to say what the feature does, the example is just supposed to show it. So could you remove this line?

Reply:
Done!